assets/cubyz/blocks/stone.zig.zon

```diff
@@ -17,6 +17,12 @@
 				0xff444444, 0xff626262, 0xff7A7876, 0xff878583, 0xff9c9996,
 			},
 		},
+		.modifiers = .{
```

Comment:
This only has an effect if you put it inside the material.

Also: If regular is fragile, sandstone definitely deserves the same fate.